From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: iusmac <iusico.maxim@libero.it>
Date: Tue, 1 Apr 2025 12:40:22 +0800
Subject: [PATCH 07/14] config: Update DEX preopt boot image profile location

They are now moved from {fwb}/config to {fwb}/boot and split into to
files.

Signed-off-by: iusmac <iusico.maxim@libero.it>
---
 config/common.mk | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index 7b14e3e..d945ff9 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -145,7 +145,9 @@ PRODUCT_SYSTEM_SERVER_COMPILER_FILTER := speed-profile
 # Use a profile based boot image for this device. Note that this is currently a
 # generic profile and not Android Go optimized.
 PRODUCT_USE_PROFILE_FOR_BOOT_IMAGE := true
-PRODUCT_DEX_PREOPT_BOOT_IMAGE_PROFILE_LOCATION := frameworks/base/config/boot-image-profile.txt
+PRODUCT_DEX_PREOPT_BOOT_IMAGE_PROFILE_LOCATION := \
+    frameworks/base/boot/boot-image-profile.txt \
+    frameworks/base/boot/boot-image-profile-extra.txt
 
 # Dedupe VNDK libraries with identical core variants.
 TARGET_VNDK_USE_CORE_VARIANT := true
